src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -176,15 +172,15 @@
     private double saturation;
 
     /**
-     * The current experience level the citizen is on.
+     * The current experiences levels the citizen is on depending on his job.
      */
-    private int level = 0;
+    private Map<String, Integer> levels =  new HashMap<>();
 
     /**
-     * The total amount of experience the citizen has.
-     * This also includes the amount of experience within their Experience Bar.
+     * The total amount of experiences the citizen has depending on his job.
+     * This also includes the amount of experiences within their Experience Bar.
      */
-    private double experience;
+    private Map<String, Double> experiences =  new HashMap<>();
```

Comment:
may be easier to combine it in one HashMap<String, Tuple<Integer,Double>> ? And as far as I can see the hashmaps can be final

Reply:
yeah